lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -18,14 +18,16 @@ class Summary extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => Material(
-        color: Colors.white,
-        elevation: 14.0,
+        color: Colors.transparent,
+        elevation: 0.0,
```

Comment:
You don't need to use `Material` widget, I guess. It renders nothing.

Reply:
I think you missed this

Reply:
@axross you are right! I changed it.